handlers/passive.js

```diff
@@ -0,0 +1,53 @@
+const log4js  = require('log4js');
+const LOG     = log4js.getLogger('passive');
+const format  = require('format');
+const _       = require('lodash');
+
+module.exports = {
+  name: 'passive',
+  command: 'passive',
+  handler: passive
+};
+
+// Having fun with messages without explicitly asking for them
+
+function passive(channel, message, client) {
+  // Storing message contents, storing lowercase version for testing against
+  let str = message.content;
+  let strTest = _.toLower(str);
+
+  // Regex functions that we'll use like a switch
+  let dadTest = new RegExp("i'm| im ");
```

Comment:
Why the spaces around the "im"?

Reply:
Has to determine if the user is identifying themselves or if its just the string "im" at the beginning or end of a word